.gitignore

```diff
@@ -1,2 +1,3 @@
 .build*
 .idea/
+.claude/
```

Comment:
there's also global gitignore files, in case you haven't used [them](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)

also, another end of file missing line.